### DISCUSSION TITLE
xgterm issues

### DISCUSSION THREAD

**Discussion by mgregg128** [iraf-community/iraf#320] [Q&A]
My xgterm windows have some problems:
1) they do not recognized ctrl-c or z, so it is impossible to abort or suspend a task.
2) perhaps a related issue - the red button in the upper left corner does not function at all.  The yellow and green buttons have the usual effect.

I've tried recompiling/installing from a new download; the problem persists.
I'm running MacOSX Ventura on a 2017 MBP.

Any suggestions on how to fix this?
Thank you,
Michael

**Comment by olebole** [comment]
There is already an open issue for this as https://github.com/iraf-community/x11iraf/issues/53. Please discuss workaround and possible solutions there.
